src/components/NavBar/DropDownItem.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
```

Comment:
You did not also update the name of the component file to match the name of the component? I still see the file named as `DropDownItem.jsx`